packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -566,3 +859,57 @@ class ReusePipelineStack extends cdk.Stack {
     });
   }
 }
+
+const createPipeline = (scope: Construct, pipelineName?: string, pipelineId: string = 'Pipeline') => {
+  const artifact = new codepipeline.Artifact();
+  return new codepipeline.Pipeline(scope, pipelineId, {
+    pipelineName,
+    crossAccountKeys: true,
+    reuseCrossRegionSupportStacks: false,
+    stages: [
+      {
+        stageName: 'Source',
+        actions: [new FakeSourceAction({ actionName: 'Source', output: artifact })],
+      },
+      {
+        stageName: 'Build',
+        actions: [new FakeBuildAction({ actionName: 'Build', input: artifact })],
+      },
+    ],
+  });
+};
+
+interface CreatePipelineStackOptions {
+  context: cdk.App,
+  suffix: string,
+  stackId?: string,
+  pipelineId?: string,
+  undefinedStackName?: boolean,
+  nestedStackId?: string,
+}
+
+const createPipelineStack = (options: CreatePipelineStackOptions): PipelineStack => {
```

Comment:
Can you actually use the `function` keyword for these? I feel like they're a little easier to discern from simple variable declarations that way.

Reply:
Sure. I tend to like this notation better but you are right that it's easier to discern with the function keyword.